.gitignore

```diff
@@ -27,7 +27,7 @@ src/clr-angular/package.json
 
 # IDE - VSCode
 .vscode/*
-# !.vscode/settings.json
+!.vscode/settings.json
```

Comment:
Line 29 makes all files inside the .vscode file to be ignored, 30 makes it so settings.json is not ignored, and the IDE takes the local TypeScript installation instead of the one bundled with VSCode. This makes TypeScript related functionality consistent between all VSCode developers.

In old versions of VSCode, things like `keyof typeof ChartShapes` would be highlighted as an error, so this setting enforces that everyone is using the correct version of TypeScript. Is a common good practice with VSCode+TypeScript to set the tsdk to be the local installation.